### PR TITLE
refactor (esl-footnotes): renaming attribute for anchor media query

### PR DESCRIPTION
**Pull request by dshovchko**

### CODE REVIEW

File: src/modules/esl-footnotes/core/esl-note.ts [hunk 1]
```diff
@@ -306,12 +309,12 @@ export class ESLNote extends ESLBaseElement {
...
 
-  /** Actions on print version changing */
+  /** Actions on anchors media query changing */
   @listen({
     event: 'change',
-    target: (el: ESLNote) => el.queryToPrint
+    target: (el: ESLNote) => el.queryToAnchor
   })
-  protected _onPrintChange(): void {
+  protected _onAnchorChange(): void {
```
[the remark this answers is not in the document]
Comment (ala-n) [reply src/modules/esl-footnotes/core/esl-note.ts:317]: same as `_onBPChange` -> `_onIgnoreConditionChange`

File: src/modules/esl-footnotes/core/esl-note.ts [hunk 2]
```diff
@@ -151,11 +154,11 @@ export class ESLNote extends ESLBaseElement {
     this._onBPChange();
   }
 
-  /** Revises the settings for the print version of note */
-  public updatePrintedQuery(): void {
-    memoize.clear(this, 'queryToPrint');
-    this.$$on(this._onPrintChange);
-    this._onPrintChange();
+  /** Revises the settings for displaying anchor inside of a note */
+  public updateAnchoredQuery(): void {
```
Comment (ala-n) [inline src/modules/esl-footnotes/core/esl-note.ts:158]:
```suggestion
  public updateAnchorQuery(): void {
```
as technically it's an anchor **'s** query

File: src/modules/esl-footnotes/core/esl-note.ts [hunk 3]
```diff
@@ -110,11 +113,11 @@ export class ESLNote extends ESLBaseElement {
     return ESLMediaQuery.for(ignore);
   }
 
-  /** Query to describe conditions to display print version of note */
+  /** Query to describe conditions to display anchor inside of a note */
   @memoize()
-  public get queryToPrint(): IMediaQueryCondition {
-    const print = this.getClosestRelatedAttr('print') || this.print;
-    return ESLMediaQuery.for(print);
+  public get queryToAnchor(): IMediaQueryCondition {
```
Comment (ala-n) [inline src/modules/esl-footnotes/core/esl-note.ts:118]:
```suggestion
  public get anchorQuery(): IMediaQueryCondition {
```

**Review by NastaLeo — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

**Pull request merged**